main.cpp

```diff
@@ -108,9 +109,46 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Limb 
+{
+    void stepForward(){}
+    int stepSize(int strideLength)
+    {
+        return strideLength;
+    }
+};
 
+struct Person 
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
 
+    void run(int howFast, int backFootLocation, int frontFootLocation, bool startWithLeftFoot);
```

Comment:
This is not what is written in the video for the `run` function.
The reason I ask people to copy this code in to this project and provide implementations for the `stepForward/stepSize` functions is to make sure they watched the video.
- [ ] revise: copy the code exactly from the video